source/storage/apis-1.5.rst

```diff
@@ -432,6 +508,47 @@ Request Headers
     response will be returned.
 
 
+**X-Weave-Records**
+
+    This header may be sent with multi-record uploads, to indicate the
```

Comment:
The non-batch ones, I'd be happy to accept for non-batch uploads in order to trigger the fail-the-entire-request behaviour.  Right now, we'll accept requests with say 200 items, save the first 100 of them, and tell you that the others failed.